src/components/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,50 @@
+import { Link, useParams } from 'react-router-dom';
+import { useMemo } from 'react';
+import classNames from 'classnames';
+
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+type TabsParams = {
+  tabId: string,
+};
+
+export const TabsPage: React.FC = () => {
+  const { tabId } = useParams<TabsParams>();
+
+  const selectedTab = useMemo(
+    () => tabs.find(tab => tab.id === tabId),
+    [tabId, tabs],
```

Comment:
```suggestion
    [tabId],
```

You don't need to add external dependencies to this array